src/main/java/uk/gov/hmcts/reform/roleassignment/controller/endpoints/GetAssignmentController.java

```diff
@@ -64,7 +64,7 @@ public GetAssignmentController(@Autowired RetrieveRoleAssignmentOrchestrator ret
                                  required = false) String correlationId,
                                 @RequestHeader(value = "If-None-Match", required = false) String ifNoneMatch,
                                  @ApiParam(value = "Actor Id ", required = true)
-                       @PathVariable("actorId") String actorId) {
+                       @PathVariable("actorId") String actorId) throws InterruptedException {
```

Comment:
I think, we shouldn't add throws exception on the controller..Please check for another solution to kill the tread in this case.